R-package/R/plot_results.r

```diff
@@ -99,14 +98,14 @@ plot_Results <- function(inPath, color = NULL) {
     ##3.b.2##
     stats <- vector("numeric", length = length(files))
     for (i in 1:length(files)) {
-        dummy <- raster::raster(files[i])
+        dummy <- raster(files[i])
         dummy[dummy < thres[i]] <- NA
         dummy[dummy >= thres[i]] <- 1
-        stats[i] <- raster::freq(dummy, value = 1, useNA = "no")
+        stats[i] <- freq(dummy, value = 1, useNA = "no")
     }
 
-    dummy <- raster::Which(!is.na(raster(files[1])))
-    ref <- raster::freq(dummy, value = 1, useNA = "no")
+    dummy <- Which(!is.na(raster(files[1])))
+    ref <- freq(dummy, value = 1, useNA = "no")
     percent <- round((stats / ref) * 100, 2)
     rest <- round(100 - sum(percent), 2)
     percent <- append(percent, rest)
```

Comment:
why is the form raster::...removed?

Reply:
Something went wrong with the checks using Rstudio.
There were also other package name missing. All fixed with 
2c0da91